DB/api/detect_events.py

```diff
@@ -0,0 +1,112 @@
+from itertools import cycle
+
+
+def detect(time_series, threshold):
+	"""
+	take in a series of time and power readings and return a list of events
```

Comment:
The summary typically shouldn't be a restatement of the params and return fields.